CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+### Fixed
+
+- Compatibility with Visual Studio 2019 (#448)
+
```

Comment:
This is the source of the current conflict, as I've pushed an update commit in the meantime.

Reply:
oops